### ISSUE TITLE
No accessible name for RRCHNM logo

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#70]
This element has no HTML markup that would provide an accessible name for the element. A screen reader would either read nothing for this element or may make a guess. Images can use the alt attribute.

**Comment by kimisgold** [comment]
Hmm, my understanding here is that the logo is using the image as a background, but it actually contains text reading "Roy Rosenzweig Center for HIstory & New Media". I'm looking at [this article on accessible names](https://developer.paciellogroup.com/blog/2017/04/what-is-an-accessible-name/) as a reference, and I would have thought this falls under using the element's content. I have similar questions about other related issues, but let's start with this one.

**Comment by IdaAkiwumi** [comment]
I think this has to do with the **aria-label** or **[aria-labelledby](https://www.w3.org/TR/WCAG20-TECHS/ARIA7.html)** and **[title*](https://www.w3.org/TR/WCAG20-TECHS/H64.html)* attributes as they are not showing any names in them. From this [website](https://www.ssa.gov/accessibility/andi/help/developerguide.html#Namers), "All interactive elements on a web page must have explicitly defined text to distinguish the element, or else the screen reader software will not know what to speak, thus the element will not be accessible to screen reader users. This distinguishing text is called the accessible name of the element.

In addition to the required accessible name, interactive elements may be given text that further describes the element or provides more information. This supporting description text is called the accessible description of the element."

I believe what is there now is probably more of a description of the content than the name/title. Also, this accessible name issue was geared toward the **[iframe](https://webaim.org/techniques/frames/)** on the logo according to ANDI. Here are a couple screenshots from Chrome Developer tools and ANDI for more info. 

![Screen Shot 2020-09-28 at 9 59 20 AM](https://user-images.githubusercontent.com/618307/94443895-b84f3e80-0173-11eb-9575-a8cb2e0768dc.png)
![Screen Shot 2020-09-28 at 10 07 53 AM](https://user-images.githubusercontent.com/618307/94443908-bb4a2f00-0173-11eb-97df-d3b3afb9251c.png)

**Comment by kimisgold** [comment]
That screenshot helps a lot. It looks like the logo itself passes muster, as it's catching the "Contents", but the accessibility check is catching the iframe used by the recaptcha badge. 

That's markup from our contact form plugin's recaptcha feature provided by Google. Because we can't control the badge markup, I've hidden the badge and included the alternative recommended copy [from their documentation](https://developers.google.com/recaptcha/docs/faq) on the relevant contact form. The copy should be much more straightforward accessibility-wise.
